samples/MegaDrone/src/main/cpp/MegaDroneEngine.h

```diff
@@ -40,15 +40,16 @@ class MegaDroneEngine : public IRestartable {
     // from IRestartable
     virtual void restart() override;
 
+    bool start();
+    bool stop();
```

Comment:
```suggestion
    void stop();
```

Reply:
thank you for bring this one up, this is just for the interface consistence purpose with regarding to  ` bool start();`